File: C/No_1.c
/* Suhaimee Binyoh
Insertion Sort Advanced Analysis
web : https://www.hackerrank.com/challenges/insertion-sort/problem
score : 06.58
*/
int insertionSort(int arr_count, int* arr) {
    long int i,j,count=0;
    for(i=1;i < arr_count;i++){
        for(j=i;j>0;j--){
            if(*(arr + j)< *(arr + j-1)){
                *(arr + j) = *(arr + j) + *(arr + j-1);
                *(arr + j -1) = *(arr + j) - *(arr + j-1);
                *(arr + j) = *(arr + j) - *(arr + j-1);
                count++;
            }
           else{
               break;
           }
        }
    }
    return count;
}

File: JavaScript/No1.js
/* Suhaimee Binyoh
Number Line Jumps
web : https://www.hackerrank.com/challenges/kangaroo/problem

*/
function kangaroo(x1, v1, x2, v2) {
    // Write your code here
    if(((x1-x2)%(v2-v1) === 0 ) && ( v1>v2)){
        console.log("YES")
        return "YES"   
    }
    else{
        console.log("NO")
        return "NO"
    }
}